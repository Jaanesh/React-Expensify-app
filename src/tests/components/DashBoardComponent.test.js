import React from 'react';
import {shallow} from 'enzyme';
import DashboardComponentTemplate from '../../components/DashBoardComponent.js';

test('should render DashBoardComponent',()=>{
    const wrapper=shallow(<DashboardComponentTemplate/>);
    expect(wrapper).toMatchSnapshot();
});