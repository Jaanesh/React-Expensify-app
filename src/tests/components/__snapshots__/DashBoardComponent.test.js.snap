// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashBoardComponent 1`] = `
<div>
  <h2>
    This is dashboard
  </h2>
  <Connect(TotalExpenseComponent) />
  <Connect(ExpenseList) />
</div>
`;
